projects/Mallard/src/components/article/types/article.tsx

```diff
@@ -103,7 +103,7 @@ const ArticleWebViewAndroid = ({
 
     useEffect(() => {
         onTopPositionChange(false)
-    }, [])
+    }, [onTopPositionChange])
```

Comment:
@RichieAHB I'm assuming we don't want this changed in and can add a line to stop the linter doing this?

Reply:
when linting goes wrong 💥